src/main/java/frc/robot/RobotContainer.java

```diff
@@ -48,6 +48,7 @@ public class RobotContainer {
   /** mailboxBelts object for the MailboxBelts class */
   public static MailboxBelts mailboxBelts = new MailboxBelts();
 
+  /** intake object for the Intake class */
```

Comment:
My dude, this is not descriptive. I would do something like the JavaDoc that we have for `RobotContainer.drive`